code/game/gamemodes/wizard/spellbook.dm

```diff
@@ -14,11 +14,13 @@
 
 /datum/spellbook_entry/proc/IsSpellAvailable() // For config prefs / gamemode restrictions - these are round applied
 	return 1
-/datum/spellbook_entry/proc/CanBuy(var/mob/living/carbon/human/user,var/obj/item/spellbook/book) // Specific circumstances
+
+/datum/spellbook_entry/proc/CanBuy(var/mob/living/carbon/human/user, var/obj/item/spellbook/book) // Specific circumstances

```

Comment:
if you're going to clean up spacing, then clean up the implied `var`.

Reply:
cleaned up all the implied vars I could find in the file